docs/tools/conviso-sbom/conviso-sbom.md

```diff
@@ -0,0 +1,111 @@
+---
+id: conviso-sbom
+title: Conviso SBOM
+sidebar_label: Conviso SBOM
+---
+
+## Introduction
+
+Today, it is almost impossible to develop software without using third-party libraries. Many solutions have already been designed and transformed into libraries that we can use and adapt in our software. However, this reliance on external libraries poses risks to our applications, as these third-party components may contain vulnerabilities and/or have licenses that conflict with our own applications.
+
+A Software Bill of Materials (SBOM) helps us build an inventory of our components, providing us with the necessary visibility into what components are included in our software. By utilizing an SBOM, organizations can:
+
+1. **Identify Vulnerabilities**: Organizations can easy see what components has vulnerabilities, quantity and cves related.
+
+2. **Ensure Compliance**: Helps ensure compliance with licensing requirements and can help avoid legal complications.
+
+3. **Streamline Audits**: Simplifies the audit process by providing a clear and comprehensive list of software components.
+
+4. **Enhance Supply Chain Security**: Organizations can better assess the security of their software supply chain and take action against potential risks.
+
+5. **Facilitate Incident Response**: Provides critical information to identify affected components quickly, allowing for a more efficient response.
+
+This is crucial for enhancing software transparency, security, and compliance. By providing visibility into the existence of vulnerabilities for each component, an SBOM aids in decision-making and prioritization.
+
+You can always learn more in our blog:
+- [Software bill of materials what it is and how it works](https://blog.convisoappsec.com/en/software-bill-of-materials-sbom-what-it-is-and-how-it-works/) to learn more about SBOM.
+- [The importance of supply chain to application security](https://blog.convisoappsec.com/en/the-importance-of-supply-chain-to-application-security/) to learn more about Supply Chain.
+
+## SBOM Formats Overview
+
+SBOMs (Software Bill of Materials) describe the components, dependencies, and metadata of software. Below are the key formats:
+
+- **CycloneDX**: Lightweight format for application security and risk management. Supports XML and JSON formats. 
+- **SPDX**: Focuses on license compliance and component tracking. Supports multiple formats like JSON, YAML, and RDF/XML.
+- **SWID**: ISO standard for identifying software assets installed on systems. Uses XML.
+- **VEX**: Focuses on vulnerability exploitability data within SBOMs, helping prioritize vulnerabilities.
```

Comment:
VEX não é um formato de SBOM. É um complemento/advisor para priorização de vulnerabilidades a partir do SBOM como input.

Reply:
fiquei um pouco confuso, o proprio CycloneDX coloca o vex como podendo ser um formato separado.
https://github.com/CycloneDX/bom-examples/tree/master/VEX
```VEX information can be represented inside an existing BOM, or in a dedicated VEX BOM```

Reply:
por ele poder ser gerado separado, ou seja, por mais que dependa de um sbom (preferencialmente no formato CycloneDX) para ter a informacao completa, ele possui um formato proprio conhecido como VEX BOM, ai mantive ele ali na listagem, mas deixando claro que ele e um complemento ao sbom, apesar de ser possivel usar somante ele. Eu to usando esse material como referencia.
https://github.com/CycloneDX/bom-examples/tree/master/VEX